### DISCUSSION TITLE
✨ Inline folding for language service

### DISCUSSION THREAD

**Discussion by itsezc** [master-co/css#389] [Ideas]
### Description

Currently, Master can create very long classes so it'd be nice if the extension supporting inline folding and also expose a regex option for this to support non-mainstream frameworks and languages too.

**Comment by itsezc** [comment]
Has there been any updates on this feature?

**Comment by 1aron** [comment]
@itsezc Not on the priority list yet, but I might find what you're looking for https://marketplace.visualstudio.com/items?itemName=AKROSS.attributes-collapse

![img](https://github.com/akr-oss/attributes-collapse/raw/HEAD/static/attribute-collapse.gif)

**Comment by itsezc** [comment]
> @itsezc Not on the priority list yet, but I might find what you're looking for https://marketplace.visualstudio.com/items?itemName=AKROSS.attributes-collapse

This is an amazing alternative for the time being, but would love for this functionality to be built into the MasterCSS VS plugin for improved DX
